### PR TITLE
chore: adding peer dependency with algosdk

### PR DESCRIPTION
**Pull request by negar-abbasi**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -24,6 +24,22 @@ This library can be installed from NPM using your favourite npm client, e.g.:
...
+```
+npx check-peer-dependencies --install
+```
+
+- Alternatively, simply use:
+
+```
+npm install algosdk
+```
+
```
Comment (neilcampbell) [inline docs/README.md:42]:
```suggestion
```
```
```
```
```

I think simply saying "Ensure you have it installed in your project" is enough here.

**Review by neilcampbell — APPROVED** [review]

**Comment by robdmoore** [comment]
I think this is probaby a breaking change? Should we add that in so major version gets bumped?

**Comment by robdmoore** [comment]
Also, let's pin it to the latest algosdk - `2.5.0`

**Comment by neilcampbell** [comment]
@robdmoore if we pin to that specific version, we will be forcing consumers to use that exact version in their projects. Assuming you're simply suggesting we update the package.json to `"algosdk": "^2.5.0"`

**Comment by robdmoore** [comment]
Yep! `^2.5.0`

**Pull request merged**